app/controllers/albums.js

```diff
@@ -0,0 +1,7 @@
+const { getAlbums } = require('../services/albums');
+
+exports.getAlbums = (_, res, next) => {
+  getAlbums()
+    .then(albums => res.send({ albums }))
```

Comment:
normally when you return a list of resources you just return a list, no need for it to be inside an empty object